test/feedback/actions/feedback.unit.spec.js

```diff
@@ -0,0 +1,86 @@
+import chai from 'chai';
+import sinon from 'sinon';
+import {
+  sendFeedback,
+  clearError,
+  SEND_FEEDBACK,
+  FEEDBACK_RECEIVED,
+  FEEDBACK_ERROR,
+  CLEAR_FEEDBACK_ERROR
+} from '../../../src/js/feedback/actions';
+
+describe('sendFeedback', () => {
+
+  const dispatch = sinon.spy();
+  const old = { sessionStorage: global.sessionStorage, fetch: global.fetch };
+  const fetchResponse = {
+    ok: true,
+    json() {},
+    headers: {
+      get: key => ({ 'content-type': 'application/json' }[key])
+    }
+  };
+  const fetch = sinon.spy(() => Promise.resolve(fetchResponse));
+
+  before(() => {
+    global.sessionStorage = {};
+    global.fetch = fetch;
+  });
+
+  after(() => {
+    global.fetch = old.fetch;
+    global.sessionStorage = old.sessionStorage;
+  });
+
+  afterEach(() => {
+    fetch.reset();
+    dispatch.reset();
+  });
+
+  it('dispatches SEND_FEEDBACK and FEEDBACK_RECEIVED when there is a successful request', (done) => {
+
+    const actionCreator = sendFeedback({ description: 'My feedback description', email: 'test@test.com' });
+    const result = actionCreator(dispatch);
+
+    result.then(() => {
+
+      chai.assert.isTrue(fetch.calledOnce, 'Fetch was called');
```

Comment:
We use the `expect` syntax for chai almost exclusively (in unit tests). Might be good to switch to that for consistency.

Reply:
Okay, I can switch to that, but ahhh, I so much prefer the traditional assert syntax.